app/services/bus_logic/hwpx2pdf.py

```
@@ -0,0 +1,122 @@
+                    
+                except Exception as action_error:
+                    print(f"  HAction도 실패: {action_error}")
+                    
+                    # 방법 3: 마지막 시도
+                    try:
+                        hwp.HAction.GetDefault("FileSaveAsPdf", hwp.HParameterSet.HFileOpenSave.HSet)
+                        hwp.HParameterSet.HFileOpenSave.filename = pdf_path
+                        hwp.HParameterSet.HFileOpenSave.Format = "PDF"
+                        hwp.HAction.Execute("FileSaveAsPdf", hwp.HParameterSet.HFileOpenSave.HSet)
+                        time.sleep(1)
+                    except:
```

Ruff (0.15.1)

[error] Do not use bare `except`

(E722)